### DISCUSSION TITLE
bacularis ui for bareos

### DISCUSSION THREAD

**Discussion by awsmaythem** [bacularis/bacularis-app#18] [General]
Hi,

Is there a Bacularis web interface for Bareos?

In the Bacularis web interface for Bacula, it is easy to add clients, jobs, and files to back up. However, in the Bareos web interface, you can only add jobs. You cannot add clients and files, except by manually adding them to the /etc/bareos/bareos-dir.d/client/example file.

Can you please help?

**Comment by ganiuszka** [comment]
Hello @awsmaythem

Thanks for your message and for feedback about using Bacularis.

It is interesting question for which I will try to answer the best as I can.

As you correctly noticed Bacularis is a web interface for Bacula. It has been designed and created for Bacula starting from first line of code. I have to admit I have never tried Bareos and I don't know almost anything about its capabilities. For this reason it is difficult for me to speak about a tool that I don't know.

For Bacularis we don't plan to extend support for other programs outside Bacula project. I would like to help you but I don't have a quick solution for your question. The only way, that I see for now, is to use Bacula with Bacularis.

Best regards,
Marcin Haba (gani)
